conf/routes

```diff
@@ -83,6 +83,10 @@ POST           /api/oauth2/auth                                             org.
 POST           /api/oauth2/token                                            org.iatoki.judgels.jophiel.controllers.api.oauth2.OAuth2APIController.token()
 GET            /api/oauth2/userinfo                                         org.iatoki.judgels.jophiel.controllers.api.oauth2.OAuth2APIController.userInfo()
 
+# Auth API
+POST          /api/auth                                                     org.iatoki.judgels.jophiel.controllers.api.internal.InternalAuthApiController.postApiLogin()
```

Comment:
I think we should prefix the new API for Frontend by /api/v2/ so we can just say "we support only API v2". This is because I see there is already /api/ and /api/v1/ usages.